pages/index.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <ModalAddNewEvent :date="date" @closeModal="closeModal" v-if="modal" />
+  <div class="root-container">
+    <Calender
+      @openModal="handleModal"
+      @toggleModal="toggleModal"
+      :Allevents="userCreatedEvent"
+      :key="update_component"
+      :selectedEvents="selectedEvents"
+    />
+    <!-- calenderRightbar -->
+    <EventNavBar :events="events" @include="addEvent" @exclude="removeEvent" />
```

Comment:
we can name events the same as the function call like `addEvent` or `removeEvent`